wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -378,10 +379,15 @@ static public boolean isWorkunitComplete(WUState state)
      */
     public static WUState getStateID(WorkunitInfo wu)
     {
-        if (wu != null)
-            return getStateID(wu.getStateID());
-        else
-            return WUState.UNKNOWN;
+        if (wu != null) {
+            if (wu.getStateID() != null) {
```

Comment:
http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html#449

https://google.github.io/styleguide/javaguide.html#s4.1-braces

I'll change the brace format, but most Java codebases use those standards.  As for squashing, that can be done from github when you merge, but I'll close this PR and submit a new one.